OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/Preferences.java

```diff
@@ -131,6 +125,76 @@ public void setFirstTime(boolean value) {
         editor.commit();
     }
 
+    public boolean isUsingEncryptedKeyRings() {
+        return mSharedPreferences.getBoolean(Constants.Pref.USING_ENCRYPTED_KEYRINGS, true);
+    }
+
+    public void setUsingEncryptedKeyRings(boolean value) {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putBoolean(Constants.Pref.USING_ENCRYPTED_KEYRINGS, value);
+        editor.commit();
+    }
+
+    public boolean isAppLockReady() {
```

Comment:
Again, a comment explaining what it does (as opposed to the `useApplock` preference - add a comment there too) would be helpful.